e2e/app/page/workspace-base.ts

```diff
@@ -156,7 +156,6 @@ export default abstract class WorkspaceBase extends AuthenticatedPage {
     await waitWhileLoading(this.page);
 
     console.log(`Deleted ${resourceType} "${resourceName}"`);
-    await this.waitForLoad();
```

Comment:
This is a bug. `workspace-base` page does not have `isLoaded` function which is called in `waitForLoad` function.